e2e2/test/cases/nvidia/manifests/mpi-job-pytorch-training-multi-node.yaml

```diff
@@ -55,11 +50,13 @@ spec:
             - RDMAV_FORK_SAFE=1
             - -x
             - NCCL_PROTO=simple
-            - -x 
+            - -x
             - FI_LOG_LEVEL=warn
             - -x
             - FI_EFA_USE_DEVICE_RDMA=1
             - -x
+            - OFI_NCCL_DISABLE_GDR_REQUIRED_CHECK=0
```

Comment:
This is important, it will fail the test case if something goes wrong with the EFA RDMA instead of falling back to the dirt-slow host buffer copy.